src/elm/Main.elm

```diff
@@ -73,6 +79,9 @@ subscriptions model =
         [ Time.every 100 Tick -- 10 times per second
         , InView.subscriptions InViewMsg model.inView
         , onScroll OnScroll
+        , Pile.subscriptions Pile1 model.pile1
```

Comment:
I wonder if this could be a list of Piles - with Section Ids to follow the pattern of other content?